test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2139,12 +2142,19 @@ class test_ucp_sockaddr_protocols : public test_ucp_sockaddr {
                 sreq = ucp_tag_send_nbx(sender().ep(), &send_buf[0], size, 0,
                                         &send_param);
             }
-            reqs.push_back(sreq);
 
             if (!is_exp) {
                 rreq = do_unexp_recv(recv_buf, size, sreq, send_stop,
                                      recv_stop);
+                reqs.push_back(sreq);
                 reqs.push_back(rreq);
+            } else {
+                ucs_status_t send_status = request_wait(sreq, 0, false,
+                                                        UCS_LOG_LEVEL_INFO);
+                if (!check_send_status(send_status, receiver(), NULL,
```

Comment:
same here - since do not have recv request, maybe just break if send_status is UNREACHABLE?